s3recovery/s3recovery/s3recovercorruption.py

```diff
@@ -31,6 +31,33 @@ def __init__(self):
         super(S3RecoverCorruption, self).create_logger_directory()
         super(S3RecoverCorruption, self).create_logger("S3RecoverCorruption")
 
+    def cleanup_bucket_list_entries(self, index_id):
+        """
+        Performs cleanup of stale data entries
+
+        """
+        list_index_response = self.list_index(index_id)
+        data_as_dict = self.parse_index_list_response(list_index_response)
+        key_list = list(data_as_dict.keys())
+        for key in key_list:
+            if key not in self.common_keys:
+                status, response = self.kv_api.delete(index_id, key)
+                super(S3RecoverCorruption, self).check_response(status, "delete", response, index_id, key)
```

Comment:
we can return the status and response from this function, to help caller to check the status, if needed ?

Reply:
Return is not required as we check status and response as a part of the check_response call.